### ISSUE TITLE
Stuck to : " INFO Loading settings from "

### ISSUE THREAD

**Issue by kemilall** [bmaltais/kohya_ss#2984]
Hey, I'am trying to train sdxl on kohya. I make correctly all my hyperparameters, my dataset seams correct also and when i press start, it loads and stay at " INFO Loading settings from "

I am a beginner in this field, thanks !

here is the logs:

09:54:54-616808 INFO     Loading config...                                                          
09:54:55-195190 INFO     SDXL model selected. Setting sdxl parameters                               
09:54:59-389411 INFO     Start training Dreambooth...                                               
09:54:59-391262 INFO     Validating lr scheduler arguments...                                       
09:54:59-392492 INFO     Validating optimizer arguments...                                          
09:54:59-394075 INFO     Validating /content/drive/MyDrive/logs/log existence and writability...    
                         SUCCESS                                                                    
09:54:59-395544 INFO     Validating /content/drive/MyDrive/logs/model existence and writability...  
                         SUCCESS                                                                    
09:54:59-396703 INFO     Validating stabilityai/stable-diffusion-xl-base-1.0 existence... SUCCESS   
09:54:59-397828 INFO     Validating /content/drive/MyDrive/logs/img existence... SUCCESS            
09:54:59-398852 INFO     Headless mode, skipping verification if model already exist... if model    
                         already exist it will be overwritten...                                    
09:54:59-401245 INFO     Folder 20_franco-belge comics style LINE ART colour style: 20 repeats found
09:54:59-407658 INFO     Folder 20_franco-belge comics style LINE ART colour style: 210 images found
09:54:59-408705 INFO     Folder 20_franco-belge comics style LINE ART colour style: 210 * 20 = 4200 
                         steps                                                                      
09:54:59-409848 INFO     Regulatization factor: 1                                                   
09:54:59-410735 INFO     Total steps: 4200                                                          
09:54:59-411578 INFO     Train batch size: 5                                                        
09:54:59-412423 INFO     Gradient accumulation steps: 1                                             
09:54:59-413275 INFO     Epoch: 15                                                                  
09:54:59-414085 INFO     Max train steps: 1600                                                      
09:54:59-414912 INFO     lr_warmup_steps = 0                                                        
09:54:59-425855 INFO     Saving training config to                                                  
                         /content/drive/MyDrive/logs/model/last_20241124-095459.json...             
09:54:59-430675 INFO     Executing command: /usr/local/bin/accelerate launch --dynamo_backend no    
                         --dynamo_mode default --mixed_precision bf16 --num_processes 1             
                         --num_machines 1 --num_cpu_threads_per_process 2                           
                         /content/kohya_ss/sd-scripts/sdxl_train.py --config_file                   
                         /content/drive/MyDrive/logs/model/config_dreambooth-20241124-095459.toml   
09:54:59-432755 INFO     Command executed.                                                          
2024-11-24 09:55:09.467153: E external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:9261] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
2024-11-24 09:55:09.467206: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:607] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
2024-11-24 09:55:09.468463: E external/local_xla/xla/stream_executor/cuda/cuda_blas.cc:1515] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2024-11-24 09:55:10.454862: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-11-24 09:55:12 INFO     Loading settings from

**Comment by FruitPigFoot** [comment]
me too.

**Comment by Hezow** [comment]
Did you manage to solve it ?

**Comment by kemilall** [comment]
No, I'm now using ai-toolkit

**Comment by FruitPigFoot** [comment]
i just moved to original sd-scripts and it was okay after then
